src/browser/new-browser.ts

```diff
@@ -71,23 +72,9 @@ export class NewBrowser extends Browser {
     async init(): Promise<NewBrowser> {
         this._session = await this._createSession();
 
-        this._extendStacktrace();
-        this._addSteps();
-        this._addHistory();
-
-        await runGroup(
-            {
-                session: this._session,
-                callstack: this._callstackHistory!,
-                config: this._config,
-            },
-            "testplane: init browser",
-            async () => {
-                this._addCommands();
-                this.restoreHttpTimeout();
-                await this._setPageLoadTimeout();
-            },
-        );
```

Comment:
Why you remove it?

Reply:
Master process only creates webdriver session
All of those is not needed here, because tests are executed in workers